src/Router/Adapter.php

```diff
@@ -142,10 +134,10 @@ private function handleAcceptableException(Message $message, DataModel $dataMode
             null,
             null,
             function () use ($e, $dataModel, $message) {
-                echo "ack but due to acceptable exception {$message->getRoutingKey()}".PHP_EOL;
+                $this->logger->notice("ack but due to acceptable exception {$message->getRoutingKey()}");
 
                 if ($previous = $e->getPrevious()) {
-                    $this->logger->warning($previous->getMessage(), [
+                    $this->logger->notice($previous, [
```

Comment:
Can you delete this one please add attach the exception to the call above please ?